docs/standard/native-interop/exceptions-interoperability.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Exceptions Interoperability"
+ms.date: "01/16/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "unmanaged code, exceptions"
+  - "exceptions, unmanaged code"
+  - "interop, exceptions"
+  - "exceptions, interop"
+---
+# Working with Interop Exceptions in Unmanaged Code
+Unmanaged code exception interop is not supported in non-Windows platforms, due to the portability issues that arise, since managed code can't know how these exception mechanisms work under the covers. The Unix ABI has no definition for this and therefore, exceptions end up resulting in unpredictable behaviors and potential crashes.
```

Comment:
I would also explicitly mention: `Unmanaged code exception interop is supported on Windows platforms only.`

Reply:
Ok! Would this be added, or replacing `not supported in non-Windows platforms`? I ask because of the potential redundancy.

Reply:
It should be added.

Reply:
The doc should be explicit about both the behavior on Windows and the behavior on non-Windows.